string relation
int32[] entity_ids
